app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -43,6 +45,20 @@ def update
       end
     end
 
+    def active_users_export
+      provider_users = SupportInterface::ActiveProviderUsersExport.call
+
+      csv = CSV.generate do |rows|
+        rows << provider_users.first.keys
+
+        provider_users.each do |a|
+          rows << a.values
+        end
+      end
+
+      send_data csv, filename: "active-provider-users-#{Time.zone.today}.csv", disposition: :attachment
+    end
```

Comment:
Perhaps we can move this into `PerformanceController` so that all exports are in the same place, it might make refactoring easier in the future and you can reuse the new `to_csv` helper.

Reply:
Yea I think it makes sense that all of the CSV exports are bunched too. I'll move it across 👍